app/validators/content_type_validator.rb

```diff
@@ -0,0 +1,13 @@
+class ContentTypeValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return if !value.attached? || content_types.select { |c| value.content_type.include?(c) }.any?
```

Comment:
why not just `return if !value.attached? || content_types.include?(c)`

Reply:
:+1: